libraries/exporters/src/main/java/com/siemens/sw360/exporter/ComponentExporter.java

```diff
@@ -55,21 +94,62 @@ public int getColumns() {
         }
 
         @Override
-        public List<String> makeRow(Component component) {
-            List<String> row = new ArrayList<>(getColumns());
+        public SubTable makeRows(Component component) {
+            return extendedByReleases
```

Comment:
it might be more readeable to use an explicit `if..else...` instead of the ternary operator.